src/core/layers/sightings.py

```diff
@@ -175,7 +167,9 @@ def _init_widgets(self, layer: QgsVectorLayer) -> None:
         ]
         setup = QgsEditorWidgetSetup("ValueMap", cfg)
         layer.setEditorWidgetSetup(idx, setup)
-        layer.setDefaultValueDefinition(idx, QgsDefaultValue("'foraging'"))
+        layer.setDefaultValueDefinition(
+            idx, QgsDefaultValue("{2839923C-8B7D-419E-B84B-CA2FE9B80EC7}")
```

Comment:
Could you defined this string `"{2839923C-8B7D-419E-B84B-CA2FE9B80EC7}"` in a dedicated global variable please?

Reply:
:+1: It's done